### ISSUE TITLE
Explicit track badge

### ISSUE THREAD

**Issue by LupSpie** [vfsfitvnm/ViMusic#239]
### Feature description

Have a little badge to the side that shows whether a track is explicit or not.

### Why do you want this feature?

The main benefit of this feature IMO is the quick distinction it creates between explicit and clean versions of songs.

### Additional information

| How it currently is | What I'm suggesting |
|--|--|
|![how it is](https://user-images.githubusercontent.com/49603837/185837609-0d9881c6-efb8-4a7b-a038-64eb39835c04.jpg)|![my suggestion](https://user-images.githubusercontent.com/49603837/185837625-6b33b626-7725-4a38-ac52-9f6e1aaa5b00.png)|

_Notice how without the explicit badge it's quite hard to tell which one is the explicit/clean track at a glance_

**Comment by vfsfitvnm** [comment]
Yeah, I should have thought of this earlier. I won't change the database schema now. However, if the problem
> is the quick distinction it creates between explicit and clean versions of songs

I can certainly make it possible for the search results screen - but when the given song is stored to the database, then this information would be lost.

**Comment by LupSpie** [comment]
> but when the given song is stored to the database, then this information would be lost.

Excuse my ignorance, but what do you mean when you say this?

**Comment by vfsfitvnm** [comment]
Long story short, the explicit badge would be a thing only when searching - i.e. it won't appear in local playlists
